### ISSUE TITLE
[Bug] the bottom/right of horizontal/vertical data zoom doesn't work as expected 

### ISSUE THREAD

**Issue by Justin-ZS** [apache/echarts#20593]
### Version

5.5.0

### Link to Minimal Reproduction

https://echarts.apache.org/examples/zh/editor.html?c=pie-simple&renderer=svg&code=PYBwLglsB2AEC8tSRg2BvAULWBjYANsAE4BcsA2gEQDEAzHQJwAcAIgEJUC6ANNrGGCFIIclhw4wxCAHMZAUzKwqAQwAeEAM5V-AXz441AQQ2ax_HABMVYFeWoAmAAwBGACwBaJwHYP7qjzKzu5evi4ArAFBrp4-fgBsUVTBsWHeSSmhfswZMVkujLkhcc5FqR4OLmVZDg7VJXT1vg5uTRWRgcl5JYmdmSXpfd3NOUPFzYVj5XRObXRVU17MXgvRxctOdYtOG43bGx1rsRu9R0uh9RujZztekzfLLrP7fqtd635bD3573_4vEUuCSBLkGf2u72OfnukPOpRelSBtSRv1htxaSMOaOWDlO2IqYPxDgh_RxMNJHhmQPm9UYK1pXlR_TpTla2xZWOZXjxXJ8DJ2_PJ3TpTwZLjeXJcXzRIqZwr8bJuIs58pcPNVhMlJNVQuKdPhSoqEvlyPZFTleoqipl7QZuLtmpN2stDl1sTpVMWT3pXqcXmlmW9TgtniD1sDfqcKpCQfVMcjjvjS2qQbdfj9ot9rxTGfDeW9gKzapzfkToYzzvL0JLBth3sRWdNZ3rIfTVpr0ar9sbZbbxJrae9nubfppXpcPubE-DKeneZj087rwu4-Tq6cg4nman2dXLlb4oVs78S8Pxb3vbPleXBWPDgPE4xq4cp8fcdDj8vj-vh9dKYck51gBmz_oyoFRuB76fCuzbAT-wGDgB25AbusE_KB_xoYWWGXkhiEVAG-YAfeoEvqRUFSgSpHwRU-HDu84QrIBwSMa4_pFKx04WpxXjzjxEFjPxcb8Ym_HOvxuo8chLFMeKHGyYRESydxsl8bJ0ZScJsmibJ4myZJTG1jJbFNgxhkqSZakmRphlaSZekmQZbFjmsrHAcaSmbOxgn-mBPleQJrm-U4dkIfJXnSa4bmoWZEWKdF-7hUhVlITZEWhdkSXVv5xHPEFXkNvlxHxb5JE5e2XB6AYsAAJ4mFo5gSLAmggAQEBgAAMhA0DyI1TXNQAFsAADu5AAGYqAQmjyBYsC6FV_DWLYABaQgALb2OIEgAEbAGAggbbATjVTgAD0p2wLt-3AId3gnbAa3AAAbvIAASKjQJYBDyAAkvg0DkFQuAQMQuDfQEVUYLNJAQPI0BgOQADkL3EJAuCTYj93nbA31jQjR33dIMgDfjx16Lw_DTdI8hmFDTXQCoa29co7DyBAABW3UyLARgAIo_RDTVgDVIDM1Q20qMQgsSEtdiUBE8SBO4x2wPudBK8wiuq8wDhKyFSsFIE8TMIE4RuIEOuBIwKv7ub2sm6rjB2xEjBK-Eruq_EHstM7jAe2q6uq24HvMCr8R3Z7utBzbbiB1K4SBN4Wvih73g26yev--7SstErdAJ6r4Q2_EztexbcfhAX3gF9nsDeA7-7J8GevO0nFv-w4Edm3rMc25rgTOGHLiZwbw-q-4VsR07Rta93sDMBHocW87oJK8bOcdxHJdGzHrcq-EDsL4n--B4wUf1znp813b6-wIPgTzHbdDeB7dBuFr96B0vdeB23qt0FHSoY9iTeyLhbKOfs16B1xBHfOv8HbeCjswBubgVbOA_hnWA8Qa5ZxVi_I2p8v4q0YDgo2VcI7uFPg7besBGDQMtoXAu09HYfzoeXA2dtO6zxvjXMe8QkEe2weAgef8IiB3CFHMu48v4QI9uEOOzdVZODwRXAuLRVE5wLkIrBVdvbKIHriHOkiVbMALgUdOaidim39vENB4oB7aLoP3ceh8PbMO8BQ_cStY4D3CJ4ihpiDaCILkfLB6cG7IO8YHfhPc3ZRzVLPUhqtq4DzYbAAByd5GJ2geHDWgcmDOwPhbDB3hKo4HmroIAA

### Steps to Reproduce

see the reproduce link

### Current Behavior

When set the `bottom/right` to zero, the horizontal/vertical data zoom controller overflows and gets truncated
It looks like the [moveHandleSize](https://echarts.apache.org/zh/option.html#dataZoom-slider.moveHandleSize) is not taken into account and causes this problem.

### Expected Behavior

No overflow and truncation

### Environment

```markdown
- OS:
- Browser:
- Framework:
```

**Comment by helgasoft** [comment]
cannot replicate

**Comment by Justin-ZS** [comment]
> cannot replicate
Do you open the link?
![截屏2024-12-17 11 35 23](https://github.com/user-attachments/assets/2c369545-8b27-402d-a5e1-796b4cbe01d9)
![截屏2024-12-17 11 35 28](https://github.com/user-attachments/assets/ab90c7b1-47ce-4fd6-83a1-bae077a11b53)

**Comment by helgasoft** [comment]
yes, I can open, zoom-in horizontal to zero and vertical is not cut
![pie-simple](https://github.com/user-attachments/assets/8bdcbf86-a6dd-417a-9ef8-e65e91a11f27)

**Comment by Justin-ZS** [comment]
> yes, I can open, zoom-in horizontal to zero and vertical is not cut ![pie-simple](https://private-user-images.githubusercontent.com/13038071/396342856-8bdcbf86-a6dd-417a-9ef8-e65e91a11f27.svg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ0MTQ2NDIsIm5iZiI6MTczNDQxNDM0MiwicGF0aCI6Ii8xMzAzODA3MS8zOTYzNDI4NTYtOGJkY2JmODYtYTZkZC00MTdhLTllZjgtZTY1ZTkxYTExZjI3LnN2Zz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjE3VDA1NDU0MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWNkMGMxYTdlNGQ0NjMwMzQ2NjNhMGJjN2VkZThiNTM0ZjI4MTQ1NTkzMTU2NGYwZDYwNWY3YTRkMmMyNzhlNjYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.u9ndgVxRaqUmz_VMm0MeYqtbICxjvq0D9-Ig6xjzHHU)

Sorry for my misleading words.
the `bottom/right` is the zoom config, not the zoom operation.
![截屏2024-12-17 13 48 25](https://github.com/user-attachments/assets/bccdb83c-778f-4c0f-9c85-9db1a7e1c377)
